### PR TITLE
added sphinx doc

### PR DESCRIPTION
**Pull request by mehradans92**

### CODE REVIEW

**Comment by whitead** [comment]
Nice job @mehradans92 ! Maybe add the workflows we did in exmol/maxent to have the docs generated in GitHub actions? You should probably also add your tests to CI.

One thing I found was that you can add type annotations to your code directly (see maxent/exmol source), rather than to sphinx docstring and then you will have typed code AND generate correctly typed docstrings. 

You may also want to consider intersphinx (see maxent or dmol book for examples) to link to types/docs for maxent code or others.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
```
Comment (whitead) [inline .github/workflows/tests.yml]: Why use a matrix if you only use one python version?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,31 @@
...
+``py0`` is a python implementation of compartmental disease modeling.
+
+![](docs/source/img/py_0.gif)
 
 ## Installation
 
-The package uses Keras (Tensorflow), geopandas and networkx.
+This package uses geopandas and networkx. Make sure you have ``gdal-config`` defined in your system before installing ``py0``. Assuming you are in a linux env, to install ``gdal`` run:
+```sh
+sudo apt-add-repository ppa:ubuntugis/ubuntugis-unstable
```
Comment (whitead) [inline README.md]: Still think you should make this an optional dependency, like we do for things in hoomd-tf. People will be more likely to try if there are fewer install. Also, these instructions are not for "linux", these are for debian env. Will not work on centos, redhat, arch etc.

File: docs/source/Quick_start.rst [hunk 3]
```diff
@@ -0,0 +1,344 @@
+Quick Start
```
Comment (whitead) [inline docs/source/Quick_start.rst:1]: Great examples!

File: setup.py [hunk 4]
```diff
@@ -27,8 +27,8 @@ def read(filename):
 
     packages=find_packages(exclude=('tests',)),
 
-    install_requires=['numpy>=1.19.1', 'matplotlib', 'scipy', 'tqdm', 'tensorflow_probability==0.11.1',
-                      'maxent@git+https://github.com/ur-whitelab/maxent.git', 'pandas', 'geopandas==0.8.1', 'geoplot==0.4.1', 'networkx==2.5'],
+    install_requires=['numpy>=1.19.1', 'matplotlib', 'scipy', 'tqdm', 'tensorflow_probability==0.11.1', 'jupyter',
```
Comment (whitead) [inline setup.py]: Might want to loosen these a bit. Not sure if exactly 0.11.1 TFP is required.

File: README.md (hunk 2 above)
Comment (mehradans92) [reply README.md]: will remove geopandas as a dependency.

File: setup.py (hunk 4 above)
Comment (mehradans92) [reply setup.py]: removed version

**Review by whitead — APPROVED** [review]

**Pull request merged**